pkg/kubectl/cmd/drain.go

```diff
@@ -495,7 +495,9 @@ func (o *DrainOptions) evictPods(pods []api.Pod, policyGroupVersion string, getP
 				if err == nil {
 					break
 				} else if apierrors.IsTooManyRequests(err) {
-					time.Sleep(5 * time.Second)
+					retryInterval := 5
```

Comment:
@gyliu513 : Currently the function evictPod is only invoking /api/v1/pods/eviction api; which return only 3 possible errors (IsTooManyRequests is one of three)
You are right in future someone might add another API call to that method. Are you recommending a more generic error message?